<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(197, 138, 255, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="aplicar">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:rgb(140,76,255);
font: 10pt &quot;Franklin Gothic Demi&quot;;</string>
    </property>
    <property name="text">
     <string>APLICAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagen1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="opciones">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:rgb(210,90,255);
font: 12pt &quot;Franklin Gothic Demi&quot;;</string>
    </property>
    <property name="currentText">
     <string>Suma Clampeada RGB</string>
    </property>
    <item>
     <property name="text">
      <string>Suma Clampeada RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resta Clampeada RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suma Promediada RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resta Promediada RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suma Clampeada YIQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resta Clampeada YIQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suma Promedida YIQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resta Promediada YIQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IF-Darker</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IF-Lighter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Producto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cociente</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Agency FB&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>IMAGEN N° 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Agency FB&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>IMAGEN N° 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Agency FB&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>RESULTADO</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
